picard/browser/filelookup.py

```diff
@@ -101,7 +101,7 @@ def _search(self, type_, query, adv=False):
         if self.mbidLookup(query, type_):
             return True
         params = {
-            'limit': 25,
+            'limit': QUERY_LIMIT,
```

Comment:
This file is lacking `from picard.const import QUERY_LIMIT` at top, leading to an error